components/BestListForm.tsx

```diff
@@ -0,0 +1,126 @@
+import Image from "next/image";
+import styles from "./BestListForm.module.css";
+import medalIcon from "../public/medal_icon.svg";
+import heart from "../public/heart.svg";
+import { useEffect, useState } from "react";
+import { getArticles } from "../lib/api";
+
+interface Writer {
+  id: number;
+  nickname: string;
+}
+interface Article {
+  content: string;
+  createdAt: string;
+  id: number;
+  image: string | null;
+  likeCount: number;
+  title: string;
+  writer: Writer;
+}
+interface ArticleList {
+  list: Article[];
+}
+
+interface BestProps {
+  pageSize: number;
+  orderBy: string;
```

Comment:
현재 기준으로 orderBy는 문자열 전체가 아닌 'recent', 'like' 두 가지의 값을 가진다고 볼 수 있습니다. 
따라서, 타입을 조금 더 특정적으로 작성해 주시면 좋을 것 같아요! 
```suggestion
  orderBy: 'like' | 'recent'
```